core/templates/dev/head/components/forms/schema_editors/SchemaBasedChoicesEditorDirective.js

```diff
@@ -37,7 +37,11 @@ oppia.directive('schemaBasedChoicesEditor', [
         'schema_based_choices_editor_directive.html'),
       restrict: 'E',
       compile: NestedDirectivesRecursionTimeoutPreventionService.compile,
-      controller: ['$scope', function($scope) {
+      controller: ['$scope', '$window', function($scope, $window) {
+        if ($scope.schema().value && $scope.schema().value === 'srl' &&
```

Comment:
In what case is it not equal to 'srl'?

Reply:
I have defined value in the schema for Speech recognition to be equal to ``srl``. So, it is only equal to ``srl`` if the schema is for Speech recognition language.
https://github.com/oppia/oppia/pull/4740/files#diff-c24716e5861ee68472a4ce82b413a6a9R68
Using this value check I was disabling this schema in browsers other than Chrome.